Integrate-Kobiton.md

```diff
@@ -0,0 +1,163 @@
+
+To get desired capabilities of a Kobiton device, follow these steps below :
+
+1. Go to [https://portal.kobiton.com/login](https://portal.kobiton.com/login) and login with your Kobiton account.
+2. Click **"Devices"** at the top of the window.
+
+![Devices button](./assets/devices-button.png)
+
+3. Hover over the device you want to run your test script on, click the gear button.
+
+![Automation Settings button](./assets/gear-button.png)
+
+4. In the popup window, you can choose the language of your script, application testing type, etc. In this example, we will be using NodeJS, therefore, change `Language` section to `NodeJS`. Then, change the value of `App Type` to `Hybrid/Native from Apps`. The desired capabilities code will be generated and displayed in the right pane of the popup (marked **blue**).
+
+![Automation Settings](./assets/desired-caps.jpg)
+
+5. Write down the generated code as we will be using it later.
+
+### 3. Configuring automation test script
+
+In the `samples` folder, we have provided two sample scripts for executing automation test on Kobiton iOS and Android devices. In those provided scripts, we will be executing automation testing of `ApiDemos-debug` application on Android and `UIKitCatalog-Test-Adhoc` application on iOS.
+
+1. Open `android-app-test.js` if you want to execute test on Android or `ios-app-test.js` if on iOS.
+
+2. Replace `desiredCaps` value with the one taken above. The result should look like this
+
+```javascript
+var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                '<YOUR_APP_URL>', 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<DEVICE_NAME>',
+  platformVersion:    '<PLATFORM_VERSION>',
+  platformName:       '<PLATFORM_NAME>' 
+}
+```
+
+> More information about Desired Capabilities and its parameters can be found in https://docs.kobiton.com/automation-testing/desired-capabilities-usage/
+
+**Execution environment**
+
+As the automation testing script is written in NodeJS, which requires the execution environment to have NodeJS installed, therefore, we will be using CircleCI official latest NodeJS container `circleci/node:latest` as the execution environment.
+
+In the provided CircleCI configuration file `.circleci/config.yml`, the execution environment has already been set to using the mentioned container.
+
+```yaml
+docker:
+    - image: circleci/node:latest
+```
+
+> For more information about how to execute automation test(s) on Kobiton, you can visit:
+> - Kobiton documentation: https://docs.kobiton.com/automation-testing/automation-testing-with-kobiton/
+> - Kobiton's samples for automation test scripts in other languages: https://github.com/kobiton/samples
+
+**Modifying the configuration file to execute the automation test script**
```

Comment:
I think this part should be the first part of this session. 

```
3. Configuring automation test script

3.1 Configuration .circleCI to execute automation test script
- Execution environment
- Modifying the configuration file to execute the automation test script

3.2 Configuring automation test script
```

Remember to ask user to choose the file that they choose when they modify the configuration file